### PR TITLE
Research levels for new cybernetic arm implants + sharp for eblade implant

### PR DESCRIPTION
**Pull request by silicons**
Esword implant is now sharp, but does not have block chance or armor penetration.
All cybernetic implants now have research levels
:cl: Mekhi
tweak: Reasonable R&D levels have been added for the new reasonable cybernetic implants.
rscadd: Energy-Blade combat implants now project a sharp blade instead of a boring dull blade.
:cl:
**Why the downvotes this PR adds ORIGIN TECH to them as in they can be deconstructed for levels, this doesn't add them to R&D.**

### CODE REVIEW

**Comment by TehZombehz** [comment]
Reasonable!(TM)

**Comment by TechnoAlchemisto** [comment]
Science power creep meme

**Comment by kevinz000** [comment]
Just on time.

**Comment by GunHog** [comment]
Most Syndicate stuff has tech levels, this should not be an exception! If you die with these things installed in you, then you certainly deserved to be ripped apart and used in ethically questionable experiments!

**Comment by LaharlMontogmmery** [comment]
Tech levels seem reasonable to me.

**Comment by Shadowlight213** [comment]
@GunHog that is unless you just buy and decon it as traitor RD so you can get high levels quickly :^)

File: code/modules/surgery/organs/augments_arms.dm [hunk 1]
```diff
@@ -191,16 +191,19 @@
 	name = "arm-mounted energy blade"
 	desc = "An illegal, and highly dangerous cybernetic implant that can project a deadly blade of concentrated enregy."
 	contents = newlist(/obj/item/weapon/melee/energy/blade)
+	origin_tech = "materials=5;combat=7;biotech=3;powerstorage=2;syndicate=5"
```
Comment (Shadowlight213) [inline code/modules/surgery/organs/augments_arms.dm]: These levels feel high for something that you plan on putting in the traitor uplink.
I'd need to look through all of the origin tech for the other traitor items to compare to be certain though.

Comment (kevinz000) [inline code/modules/surgery/organs/augments_arms.dm]: Oh yeah I forgot this isn't nukeop only for now

Comment (kevinz000) [inline code/modules/surgery/organs/augments_arms.dm]: @Shadowlight213 so should i lower this

**Comment by KorPhaeron** [comment]
RD levels are a trash system and I hope someone gets rid of them all together some day

**Comment by ExcessiveUseOfCobblestone** [comment]
We could scrap tech entirely and just have it so Duke's guide in Duke's order was how you obtained items and no one would know the difference.

**Comment by callanrocks** [comment]
Change it to a timer that counts to 25 and just slowly unlocks stuff during the round. Emagged counts to 3 but gives you traitor stuffs.

_Balanced._

**Comment by kevinz000** [comment]
NEW SPRITES AND RESEARCH LEVELS NERFED, MERGE READY ;)

**Comment by PKPenguin321** [comment]
idk dude, 2x the price for a weaker eblade with the only advantage being that you can stuff it into your arm? that TC price is kinda crazy

**Comment by Shadowlight213** [comment]
@kevinz000 
>merge ready
>has conflicts

WEW

**Comment by kevinz000** [comment]
FUCK

**Pull request closed without merging**